### PR TITLE
Updated header/menu to ruuterV2 version

### PR DESCRIPTION
**Pull request by ExiRain**
- MIgrated header/menu to ruuter v2
- Added howler dependencies
- Reworked generic DSL's
- Update env variables

Related [task](https://github.com/buerokratt/Buerokratt-Chatbot/issues/403).

### CODE REVIEW

**Review by 1AhmedYasser — APPROVED** [review]

File: GUI/.env.development [hunk 1]
```diff
@@ -10,6 +8,7 @@ REACT_APP_API_PATH=/
 REACT_APP_APP_PORT=3006
 REACT_APP_LOCAL=true
 REACT_APP_API_URL=http://localhost:8086
+REACT_APP_RUUTER_PRIVATE_API_URL=http://localhost:8086
```
Comment (1AhmedYasser) [inline GUI/.env.development:11]: It should be REACT_APP_RUUTER_API_URL without private or public added

File: GUI/src/App.tsx [hunk 2]
```diff
@@ -8,9 +8,15 @@ import { UserInfo } from "./types/userInfo";
...
   if (import.meta.env.REACT_APP_LOCAL === "true") {
-    const { data } = useQuery<UserInfo>({
-      queryKey: ["custom-jwt-userinfo", "prod"],
-      onSuccess: (res: any) => useStore.getState().setUserInfo(res.response),
+    useQuery<{
+      data: { custom_jwt_userinfo: UserInfo };
+    }>({
+      queryKey: ["userinfo", "prod"],
+      onSuccess: (res: any) => {
+        console.log('USERINFO')
```
Comment (1AhmedYasser) [inline GUI/src/App.tsx:16]: Please remove logging statements

File: docker-compose.yml [hunk 3]
```diff
@@ -118,14 +118,13 @@ services:
...
       - REACT_APP_MENU_URL=https://admin.dev.buerokratt.ee
-      - REACT_APP_RUUTER_V1_PRIVATE_API_URL=http://localhost:8086
-      - REACT_APP_RUUTER_V2_PRIVATE_API_URL=http://localhost:8086
       - REACT_APP_RUUTER_V2_SERVICE_API_URL=http://localhost:8086
       - REACT_APP_AUTH_BASE_URL=http://localhost:8086
       - REACT_APP_AUTH_PATH=/auth
       - REACT_APP_MENU_PATH=/chat/menu.json
       - REACT_APP_API_PATH=/
       - REACT_APP_LOCAL=true
+      - REACT_APP_RUUTER_PRIVATE_API_URL=http://localhost:8086
```
Comment (1AhmedYasser) [inline docker-compose.yml:127]: Please modify it to be Without private or public

**Review by 1AhmedYasser — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (ExiRain) [reply docker-compose.yml]: Updated

**Review by 1AhmedYasser — APPROVED** [review]

**Review by 1AhmedYasser — APPROVED** [review]

**Pull request merged**